python/katana/_property_graph.pyx

```diff
@@ -140,13 +142,13 @@ cdef class PropertyGraphBase:
         Can be called from numba compiled code.
         """
         cdef uint64_t prev
+        #TODO(amber): need to dereference GraphTopology::edge_begin()/edge_end()
```

Comment:
@arthurp : Can I remove the line above that declares a seemingly unused variable?

Reply:
Yep. My bad.